src/Salt/SiteBundle/Controller/DefaultController.php

```diff
@@ -52,4 +60,58 @@ public function importAction(Request $request)
             'message' => 'Success'
         ]);
     }
+
+    /**
+     * @Route("/salt/case/export/{docId}", name="export_cvs_file")
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function exportExcelAction($docId, Request $request)
+    {
+        $repo = $this->getDoctrine()->getManager()->getRepository(LsDoc::class);
+        $cfDoc = $repo->findOneBy(['id' => $docId]);
```

Comment:
Since you don't check if cfDoc returns anything, why don't you just get this as a parameter instead of having the id be passed as the parameter?  Then the existance will be checked by the routing code.